pycodestyle.py

```diff
@@ -169,7 +169,7 @@ def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
         'while',
     )))
 )
-DUNDER_REGEX = re.compile(r'^__([^\s]+)__ = ')
+DUNDER_REGEX = re.compile(r"^__([^\s]+)__(:\W+[a-zA-Z.0-9_\[\]\"]+)? = ")
```

Comment:
should the `\W+` here be `\s*` ?

Reply:
Probably, I was trying to fit this in in 15m after having just woken up so the caffeine hadn't hit yet

Reply:
Updated and made it a non-capturing regex as well